### ISSUE TITLE
Since updating to 4.0 Categories in Menu not working

### ISSUE THREAD

**Issue by kb61401** [WordPress/gutenberg#11081]
I have a few items in the menu that link to categories. Since updating to 4.0, these links no longer work and instead just bring up an old version of the homepage that doesn't contain my newest post in the feed.

**Comment by designsimply** [comment]
I am not sure this is directly related to Gutenberg so I would like to ask a few questions to start.

To confirm, all of the menu links work normally except any which point to category pages such as http://example.com/category/testing/ ?

Are you able to update to the latest version of the plugin and clear your browser cache and check the category links in the site's main menu again? The latest version is currently 4.1.1.

What is your WordPress version?

What theme are you testing with and what is the version of the theme?

Have you made any modifications recently in WP Admin > Settings > Permalinks or have you made any recent changes to your web server configuration?

**Comment by kb61401** [comment]
Hi, I misspoke, I was on 4.1.0, but I've updated to 4.1.1 and I am getting the same result. I have also cleared my browser cache. Yes, all menu links work normally except any which point to category pages. The theme I'm using is Kale Pro by Lyra Themes, version 2.3 (up to date) and I'm on Wordpress version 4.9.8. I have not made any recent modifications in WP Admin > Settings > Permalinks or made any recent changes to my web server configuration. This has only started happening since the 4.0 update. Thanks!

**Comment by Pikkals** [comment]
Describe the bug
On Gutenberg 4.1 the Categories do not drop down.

To Reproduce
Steps to reproduce the behavior:

Go to 'POSTS'
Click on 'ALL POSTS'
Click on any article
Click on 'DOCUMENT'
Scroll down to 'Categories'
![image](https://user-images.githubusercontent.com/42873768/47547751-02c84080-d8f7-11e8-8dac-692bb7fbd8b6.png)
Click to drop down Categories, Categories do not drop down.

I did notice in Gutenberg 4.0 that the Categories took a while to drop down sometimes but they always did...eventually.  I do have a lot of Categories.  And I always select a Yoast Primary Category.

Expected behavior
For Categories to drop down so that they can be selected

Desktop (please complete the following information):

OS: Windows 10
Browser CHROME
VersionV 69.0.3497.100 (Official Build) (64-bit)
Additional context

Gutenberg 4.1

**Comment by annestk** [comment]
Are you by any chance using the Yoast plugin?

I noticed the [changelog](https://yoast.com/wordpress/plugins/seo/change-log-wordpress-seo/) for their latest version (9.0.2) mentions fixing a bug where the category picker was not showing up with Gutenberg.

**Comment by kb61401** [comment]
Interesting. I am also using Yoast.

**Comment by Pikkals** [comment]
Right I've sent their support and email to this issue.

**Comment by kb61401** [comment]
I just checked my plugins and YOAST was missing for some reason. I deleted the wordpress-seo folder from the plugin directory and had planned on re-installing, but I wanted to double check if this was causing the inability to link to the category menu items on my site. However, without YOAST installed, I am still receiving this issue. When I click on one of the category menu items the url changes to the category, but it displays the homepage.
